plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -0,0 +1,146 @@
+module Kubernetes
+  class ReleaseDoc < ActiveRecord::Base
+    self.table_name = 'kubernetes_release_docs'
```

Comment:
I'm looking at the documentation for [Rails Engines](http://edgeapi.rubyonrails.org/classes/Rails/Engine.html), and it appears that if I use `isolate_namespace`, my Kubernetes plugin code will be unable to access info from the main app.

> If the MyEngine engine is marked as isolated, FooController only has access to helpers from MyEngine, and url_helpers from MyEngine::Engine.routes.

So I wouldn't be able to generate links from the Kubernetes plugin to any other views (like DeployGroups), which seems like it would be limiting.